specification/iothub/resource-manager/Microsoft.Devices/stable/2020-03-01/iothub.json

```diff
@@ -2638,8 +2637,7 @@
         "name": {
           "readOnly": true,
           "type": "string",
-          "description": "The resource name.",
-          "pattern": "^(?![0-9]+$)(?!-)[a-zA-Z0-9-]{2,49}[a-zA-Z0-9]$"
```

Comment:
This has been flagged as a breaking change, but it really isn't.

1. The validation is flawed, and fails a customer request when it shouldn't. One option is updating the validation, but that leads me to #2.
1. The property is read-only. The user can't set it, so no validation is necessary.

We have a blocked customers impacted by this issue, and we'd like to push this through so we can issue an SDK update ASAP.